src/test/java/com/bobocode/session/SessionExceptionsTest.java

```diff
@@ -0,0 +1,98 @@
+package com.bobocode.session;
+
+import com.bobocode.Dialect;
+import com.bobocode.exception.EntityMappingException;
+import com.bobocode.session.dto.EntityClass;
+import com.bobocode.session.dto.NotDefinedIdField;
+import com.bobocode.session.dto.NotEntityClass;
+import org.assertj.core.api.ThrowableAssert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.sql.DataSource;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@ExtendWith(MockitoExtension.class)
+class SessionExceptionsTest {
+
+    @Mock
+    private DataSource dataSource;
+
+    @Mock
+    private Dialect dialect;
+
+    private Session session;
+
+    @BeforeEach
```

Comment:
Why should we create a new session for each test? Can replace this method to @InjectMocks annotation?What do you think?

Reply:
@StanislavZabramnyi  This class tests only find methods for now. But we will add implementation of other methods (update, insert etc).. A bunch of tests will change the data in the DB and it would be wrong to run them in the same session.
Please, share your thoughts about this.

Reply:
@StanislavZabramnyi I believe it's a good practice to setup tests the way that they are guaranteed to be isolated, even if it's not necessary for now, as @nadya-gainutsa says we'll add more methods which might be affected. 
So, in this case I guess it's better to do isolation instead of preliminary execution time optimization.

Reply:
Of course! I totally agree.👌